src/components/Authentication/Register.vue

```diff
@@ -2,6 +2,25 @@
   <div id="register">
     <div class="card-body">
       <img src="../../assets/logo.png" class="rounded mx-auto d-block img-fluid" id="falkoLogoRegister">
+    
+      <div v-if="this.email_alert" class="alert alert-danger fade show" role="alert">
+        <div class="row">
+          <div class="column alert-column-left">
+            Email has already been taken
```

Comment:
Same as above. The error message cannot be hardcoded.

Reply:
OK.

but one question:
From what I've seen, there is only one case of error handling where you would need to send the data to the backend check. That would be the email already registered. If this happens, you would need to notify the user about. So, would it be necessary to implement the notification to get a list of errors sent from the backend?

The front end is validating the other data required before the user can click the "Register" button.